cloudferry_devlab/cloudferry_devlab/tests/config.py

```diff
@@ -694,7 +698,9 @@
      'volume_type': 'nfs2',  'metadata': {'data': 'nope', 'enabled': "False"},
      'server_to_attach': 'server2', 'device': '/dev/vdb'},
     {'display_name': 'cinder_volume3', 'size': 1,
-     'user': 'test_volume_migration'}
+     'user': 'test_volume_migration'},
```

Comment:
Looks like the current networking configuration in config,py does not support when we try to assign floating IP ('fip': True) to tn1server1. Assignment of Floating IP to tn1server1 is a must to attach tn1_volume1 to it.